Patches/MechanoidsExtraordinaire/ThingDefs_Races/Races_Mechanoid_MechanoidsExtraordinaire.xml

```diff
@@ -51,7 +51,8 @@
 							<power>10</power>
 							<cooldownTime>1.35</cooldownTime>
 							<linkedBodyPartsGroup>LeftHand</linkedBodyPartsGroup>
-							<armorPenetration>0.13</armorPenetration>
+							<armorPenetrationBlunt>1.5</armorPenetrationBlunt>
+							<armorPenetrationSharp>0.75</armorPenetrationSharp>
 						</li>
 						<li Class="CombatExtended.ToolCE">
 							<label>right fist</label>
```

Comment:
For several, Blunt damage type, but Sharp AP?

Reply:
@Magichp - same as above, how did you come up with these values?

Reply:
Sorry for the late answer. These values were set by looking at other mods and the value sheets in order to make those mechanoids up-to-par with the other dangers of Rimworld.

Basicly these numbers are just my take on balancing this, since a bit of balance was required to work with CE. Feel free to change those if you need, I've never been really good at balancing things anyway ^^

Reply:
Understood, @Magichp - we'll revise these values accordingly.

Reply:
Fixed in 4686cf5

Vespa fist armor penetration now identical to vanilla Lancers (sharp AP = 0). All other mechanoids already have sharp AP = 0 for similar melee tools.